8_ephem_bot.py

```diff
@@ -12,46 +12,57 @@
   бота отвечать, в каком созвездии сегодня находится планета.
 
 """
-import logging
-
+import logging, ephem, settings
+from datetime import date
 from telegram.ext import Updater, CommandHandler, MessageHandler, Filters
-
 logging.basicConfig(format='%(name)s - %(levelname)s - %(message)s',
                     level=logging.INFO,
                     filename='bot.log')
 
+def greet_user(update, context):  
+    print('Вызван /start')
+    print(update.message.reply_text('Здравствуй пользователь!'))
 
-PROXY = {
-    'proxy_url': 'socks5://t1.learn.python.ru:1080',
-    'urllib3_proxy_kwargs': {
-        'username': 'learn',
-        'password': 'python'
-    }
-}
-
-
-def greet_user(update, context):
-    text = 'Вызван /start'
-    print(text)
-    update.message.reply_text(text)
-
+def name_planet(update, context):  
+    print('Вызван /planet')
+    print(update.message.reply_text('Введите название планеты на английском -\
+ Mercury, Venus, Mars, Jupiter, Saturn, Uranus, Neptune, Pluto'))
 
 def talk_to_me(update, context):
     user_text = update.message.text
     print(user_text)
-    update.message.reply_text(text)
-
+    
+    planets = {"Mercury": ephem.Mercury("2024/05/27"), 
+                "Venus": ephem.Venus("2024/05/27"), 
+                "Mars" : ephem.Mars("2024/05/27"), 
+                "Saturn": ephem.Saturn("2024/05/27"), 
+                "Uranus": ephem.Uranus("2024/05/27"), 
+                "Neptune": ephem.Neptune("2024/05/27"),
+                "Pluto": ephem.Pluto("2024/05/27")
+                }
```

Comment:
```suggestion
                "Pluto": ephem.Pluto
                }
                current_planet = planets[user_text](current_date)
                
```
обрати внимание что у меня "лишние" скобки в конце. это потому что из словаря мы вытаскиваем функцию! которую надо вызвать.